ddl-scripts/mssql/mssql-quartz-schema.sql

```diff
@@ -3,17 +3,15 @@ CREATE TABLE [dbo].[QRTZ_CALENDARS] (
   [SCHED_NAME] [VARCHAR] (120)  NOT NULL ,
   [CALENDAR_NAME] [VARCHAR] (200)  NOT NULL ,
   [CALENDAR] [VARBINARY] (max) NOT NULL
-) ON [PRIMARY]
-GO
+) ON [PRIMARY];
```

Comment:
Not sure if I got your point.
This changes is only to normalize all ending sql statements by using the semicolon, as this is supported by all certified databases.
Also, it will be easier to configure the maven-sql-plugin in the related [PR](https://github.com/jboss-integration/bxms-qe-tests/pull/3997) by not having to distinguish between different delimiter types depending on each database. See https://github.com/jboss-integration/bxms-qe-tests/blob/a1b601a6a811b7029f13371457d47a3699c3b17a/test-pim-service-backend/pom.xml#L141-L194

Reply:
OK, so now mssql accepts `;` as well. I thought that only option was `GO`, but that just separates batches.